relay-event-normalization/src/event.rs

```diff
@@ -767,7 +767,11 @@ fn normalize_performance_score(
                     let mut normalized_component_weight = 0.0;
                     if let Some(value) = measurements.get_value(component.measurement.as_str()) {
                         normalized_component_weight = component.weight / weight_total;
-                        let cdf = utils::calculate_cdf_score(value, component.p10, component.p50);
+                        let cdf = utils::calculate_cdf_score(
+                            value.max(0.0), // Webvitals can't be negative, but we need to clamp in case of bad data.
```

Comment:
I don't think this is enough, `utils::calculate_cdf_score` calculates `f64::ln(value)` which produces infinity for `0`.